backend/src/test/java/gov/cdc/usds/simplereport/service/PatientBulkUploadServiceTest.java

```diff
@@ -108,8 +108,8 @@ void withoutFacilityId_validCsvUploadsToOrganization() {
     // WHEN
     this._service.processPersonCSV(inputStream, null);
 
-    assertThat(getPatientsForFacility(firstFacilityId).size())
-        .isEqualTo(getPatientsForFacility(secondFacilityId).size());
+    assertThat(getPatientsForFacility(firstFacilityId))
+        .hasSameSizeAs(getPatientsForFacility(secondFacilityId));
```

Comment:
@BobanL I lost your original comment, but good catch - I was a little too willing to accept IntelliJ's suggested method 🤦 Fixed now!